### DISCUSSION TITLE
Just checking before giving money to Vimeo: there are no quota problems with Vimeo right?

### DISCUSSION THREAD

**Discussion by davegoopot** [mozilla/hubs#2954] [General Questions & Discussion] [answered]
With the quota limits on YouTube, I'm just about to pay Vimeo some money to host my content so that I can get it into Hubs (too large for the basic plan). Before I pay the money can I check: does anyone have any problems playing Vimeo host video in Hubs?

[UPDATE] I've just spotted this open ticket. Is it a worry? https://github.com/mozilla/hubs/issues/1936

**Comment by emclaren** [comment, marked as answer]
Hi @davegoopot - very good question. I've found that Vimeo is much more reliable than youtube and haven't heard any reports similar to #1936 for a long time (I also note that the video I point to as an example in #1936  actually loads properly in Hubs now). 

That being said, in our discord (https://discord.gg/wHmY4nd) a few folks have reported from time to time getting an "Unable to load " message on videos possibly hosted on Vimeo. I haven't experienced this myself with Vimeo though, and I can't say for sure if this happens & how frequently. There is also a bit of a weird Vimeo bug that doesn't have an impact how they videos load in Hubs, but makes some of them not have a preview in Spoke: https://github.com/mozilla/Spoke/issues/964 

A few alternative options which might be more reliable would be to self-host the video, for example you could save the file on an amazon s3 bucket (or other hosting site that you have management of), or if you use Hubs Cloud instead of hubs.mozilla.com you could use youtube/vimeo and they would likely be less likely to be blocked because you would likely have a better sense of who is using your hubs instance at any given time (whereas on the public version of hubs, you don't know if other people are watching videos at the same time, which could be causing us to get blocked by these other services.)

**Reply by davegoopot** [reply]
I've paid for the Vimeo "plus" account. No problems so far.

**Reply by emclaren** [reply]
Hooray! please do let us know if you experience any - I asked our team and no one reported any issues being experienced with Vimeo either, but I worry it might happen and we don't hear about it 😀

**Reply by bemyevent** [reply]
I use amazon s3 and cloudfront and it works very well . I have try with Vimeo and it works well too .

**Reply by emclaren** [reply]
this is super helpful feedback thanks @bemyevent
